src/mesh/voxel_cloud.rs

```diff
@@ -365,44 +362,30 @@ impl VoxelCloud {
                 }
             }
         }
+        let (block_start, block_dimensions) =
```

Comment:
I think you don't need to `let` this, the last expression of the function can be an `if/else` block